packages/builder/src/components/upgrade/UpgradeModal.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import { Modal, ModalContent, Button } from "@budibase/bbui"
+
+  let upgradeModal
+
+  const onConfirm = () => {
+    window.open("https://accounts.budibase.com/install", "_blank")
+  }
+</script>
+
+<div>
+  <Button
+    on:click={upgradeModal.show}
+    quiet
+    primary
+    size="M"
+    icon="DownloadFromCloud">Upgrade to self-hosted</Button
```

Comment:
I think I prefer the "launch" icon... I can't get past the fact that the down arrow contrasts too much with word "upgrade".

Reply:
@joebudi are you happy with the cloud download arrow or what do you think?

Reply:
I think the problem is not the icon but the word 'upgrade'. I prefer 'progress' or 'advance'. Happy with either.

Reply:
I agree with the "launch" icon contrasting with the wording of it. 
 
"Self-host Budibase" sounds good to me.

Reply:
Because then you can still have "Upgrade to Self-hosted" in the modal. If you're looking to have somewhere that describes that it would be an upgrade.

Reply:
![image](https://user-images.githubusercontent.com/5665926/129200825-806ab4a2-40b3-46aa-9594-d05cb7aa206b.png)

Reply:
This works for me.

Reply:
@aptkingston good with you?

Reply:
Looks good!